### PR TITLE
test

### PR DESCRIPTION
**Pull request by gdvl777**
- [x] Write unit tests with React Testing Library.

### CODE REVIEW

**Comment by topeogunleye** [comment]
@gdvl777, can you please add the note from student success

**Comment by gdvl777** [comment]
> @gdvl777, can you please add the note from student success

I have communicated with them, but I haven't received a response from Student Success. I didn't want to wait any longer because time is running out to complete all my activities for this week.

**Comment by topeogunleye** [comment]
> > @gdvl777, can you please add the note from student success
> 
> I have communicated with them, but I haven't received a response from Student Success. I didn't want to wait any longer because time is running out to complete all my activities for this week.

@gdvl777, When they reply. Attach their Note please

**Comment by topeogunleye** [comment]
Hi @gdvl777, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a pair programming project with no contributions from your partner.  Kindly report any issue you had with your partner to the `student success` team and they'll permit you to work solo or look for a suitable partner. Attach a screenshot of the message you received from `student success` stating that you can do the project alone in the pull request description and resubmit this PR👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @gdvl777, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that is pair-programming which means every member of the group must have a solid fair contributions to the PR commits. This milestone was assigned to three members only two people contributed. You need to receive a go ahead from student success before you can work solo. Kindly provide evidence from SS signalling to work alone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/tests/operate.test.js
Comment (LionRouge1) [inline src/tests/operate.test.js:1]: - [x] Nice work writing unit tests for the files `operate.js` and `calculate.js` using Jest :clap: However it also requested in the project requirement to create unit tests for all React components(see image). Kindly write a tests for the components to follow the project requirement.
Screenshot from 2023-09-21 21-42-23

File: README.md [hunk 1]
```diff
@@ -145,24 +150,26 @@
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Daniel Villalba Lynch**
+👤 **Daniel Villalba**
```
Comment (LionRouge1) [inline README.md:154]: - [x] According to GitHub history 3 persons contributed to this project, but we have only one person information under the **Authors** section. Kindly make sure we have all the authors listed in the **Authors** section to make your README file looks more professional.

File: README.md [hunk 2]
```diff
@@ -189,6 +196,6 @@
 
 ## 📝 License <a name="license"></a>
 
-> This project is [MIT](https://github.com/gdvl777/math_magicians/blob/events/MIT.md).
+> This project is [MIT](https://github.com/gdvl777/math_magicians/blob/develop/MIT.md).
```
Comment (LionRouge1) [inline README.md]: - [x] Please note that the license link is linked to a 404 page. Kindly make sure there is no broken link in your README file. That will show your attention to detail and your professionalism.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi **Team**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Set up React Testing Library :heavy_check_mark: 
- Write unit tests for the files `operate.js` and `calculate.js` using Jest :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bambzee** [comment]
Note for future code reviewers: Student Success **did not grant** permission to the student to work alone during this pair-programming week.

File: src/tests/quote.test.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    expect(global.fetch).toHaveBeenCalled();
+  });
+
+  // Check if the loading text is not displayed
+  expect(getByText('Loading...')).not.toBeInTheDocument();
+
+  // Check if the quote is displayed
+  const quoteElement = getByText('Test Quote');
+  expect(quoteElement).toBeInTheDocument();
+});
```
Comment (Johnadibe) [inline src/tests/quote.test.js]: > - [ ] Good job so far Team👍, While reviewing the unit test implementation, I encountered some errors that need attention. It's important to ensure that our tests are accurate and reliable to effectively assess our project. Could you please take a look at the unit tests you've written and address the errors?
> 
> err
> 
> err1
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
### Hi Team🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



### Kindly note that I am reviewing this PR because now all the students have a minimum code contribution.

 ## Highlights
- Implemented some of the changes requested by the previous reviewer✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @gdvl777, @Johnadibe,

## Highlights!
- Good code organization and tests ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**